demo/src/renderDemo.js

```diff
@@ -54,26 +54,38 @@ function getComponent(name) {
   return registry[name];
 }
 
+function Demo() {
+  const fixedHeight = 600;
+  const [isExpanded, toggleIsExpanded] = useReducer(v => !v, true);
+  return (
+    <div style={(isExpanded ? { height: `${fixedHeight}px`, width: `${fixedHeight}px` } : {})}>
+      <button onClick={toggleIsExpanded}>Toggle Expanded Grid</button>
+      <VitessceGrid
+        layout={layout}
+        getComponent={getComponent}
+        draggableHandle={`.${handleClass}`}
+        padding={50}
+        height={isExpanded}
+        rowHeight={(isExpanded ? fixedHeight/4 : undefined)} /* If undefined, will fill window height. */
```

Comment:
Is the condition here flipped?  It starts expanded, but then the `rowHeight` is defined?

Reply:
Yes, good catch. Not sure how I mixed that up 😆